### ISSUE TITLE
dependencies seem to be throwing an error

### ISSUE THREAD

**Issue by kaushikgopal** [arrowtype/recursive-code-config#13]
I was recently trying to generate another variant of the font, but running through the installation steps again, it seems like there might be an error. 

I suspect that installing the dependencies from `requirements.txt` has bumped some lib dependency which has had a breaking api change? 

Here is the command + error:

```
(venv) bash-5.1$ python3 scripts/instantiate-code-fonts.py premade-configs/config.light.cli.yaml 
// ...
Traceback (most recent call last):
  File "~/recursive-code-config/scripts/instantiate-code-fonts.py", line 226, in <module>
    splitFont()
  File "~/recursive-code-config/scripts/instantiate-code-fonts.py", line 155, in splitFont
    dlig2calt(outputPath, inplace=True)
  File "~/recursive-code-config/scripts/dlig2calt.py", line 70, in dlig2calt
    font['glyf'].setCoordinates(glyphName, newCoords, font)
  File "~/recursive-code-config/venv/lib/python3.9/site-packages/fontTools/misc/loggingTools.py", line 529, in wrapper
    return func(*args, **kwargs)
TypeError: setCoordinates() takes 3 positional arguments but 4 were given
```

This was previously working fwiw.

**Comment by kaushikgopal** [comment]
i confirmed by suspicion by downgrading `fonttools` which i suspected was throwing the error from the error stack trace. Until a fix is in, the workaround is as follows:

```
# in case you've installed a newer version of fonttools
pip uninstall fonttools
pip install fonttools==4.17.0
```

It seems like things are working with `4.17.0` of fonttools. Upgrading from there, seems to be causing issues.

**Comment by arrowtype** [comment]
Closed with the PR #14. Thanks again, Kaushik!
